Content.MapRenderer/Program.cs

```diff
@@ -104,45 +102,118 @@ internal static async Task Main(string[] args)
                 Console.WriteLine($"Selected maps: {string.Join(", ", selectedMapPrototypes)}");
             }
 
+            var maps = new List<RenderMap>();
+
             if (arguments.ArgumentsAreFileNames)
             {
                 Console.WriteLine("Retrieving maps by file names...");
+
+                //
+                // Handle legacy command line processing:
+                // Ideally, people pass file names that are relative to the process working directory.
+                // i.e. regular command-line behavior.
+                //
+                // However, the map renderer was originally written to only handle gameMap prototypes,
+                // so it would actually go through the list of prototypes and match file name arguments
+                // via a *very* coarse check.
+                //
+                // So if we have any input filenames that don't exist... we run the old behavior.
+                // Yes by the way this means a typo means spinning up an entire integration pool pair
+                // before the map renderer can report a proper failure.
+                //
+                // Note that this legacy processing is very important! The map server currently relies on it,
+                // because it wants to work with file names, but we *need* to resolve the input to a prototype
+                // to properly export viewer JSON data.
```

Comment:
This is not true. I adapted it to provide relative paths. Maybe we want to support instances running an older version of the map server but we might just as well remove the legacy behaviour

Reply:
> This is not true. I adapted it to provide relative paths.

Yes, but, as I explained on Discord, we *need* to keep using the old behavior of doing path -> prototype resolution, otherwise we cannot properly export correct map names.